src/scss/components/navigation/responsive.scss

```diff
@@ -0,0 +1,130 @@
+@media screen and (max-width: 1200px) {
+  .main-navigation__list {
+      margin-right: 0;
+  }
+}
+@media screen and (max-width: 992px) {
+  .main-navigation {
+    position: relative;
+    z-index: 100;
+    &__list {
+      position: absolute;
+      right: -400px;
```

Comment:
Recommend to either fix menu width to the same value as a shift {width: 400px; right: -400px;} or applying other techniques (e.g. {left: 200vw}, .wrapper {overflow: hidden} .inner {left: 100%;}).